Apps/Sandcastle/gallery/Model Experimental 3D Models.html

```diff
@@ -142,7 +142,7 @@
 
           model.readyPromise.then(function (model) {
             viewer.camera.lookAt(
-              position,
+              model.boundingSphere.center,
```

Comment:
The CesiumMan model was partly out of frame because the camera didn't adjust for the model's height, so I changed it for more consistent viewing.